src/models/announcement.cr

```diff
@@ -1,6 +1,7 @@
 require "granite_orm/adapter/pg"
 require "markdown"
 require "autolink"
+require "../helpers/*"
```

Comment:
We can remove this line now...

Reply:
I forgot it! We can remove this line from ```application_controller.cr``` too.